src/core/storage/Cookie.ts

```diff
@@ -28,7 +28,7 @@ export default class CookieStorage {
     }
 
     public static set(id: string, value: string, domain: string = '', path: string = '/') {
-        let expires = new Date((new Date().getTime() + __20years)).toUTCString();
+        let expires = new Date(Date.now() + __20years).toUTCString();
```

Comment:
Unrelated cleanup that was done while the issue still included persistence. Keeping this small change around.